lang/en/docs/materials/classification/non-periodic.md

```diff
@@ -0,0 +1,17 @@
+# Non-Periodic Materials
```

Comment:
So are we introducing materials classification? If yes, we should create an overview file at that level - `materials/classification/overview.md` and refer to the included content from there. Periodic Materials should also be described. And maybe a reference to materials sets.

Reply:
Created a new overview file.  https://github.com/Exabyte-io/documentation/pull/219/files#diff-85aac972bf958fff2b551016debab75eb161ff16bc3ae5ce2bc0bce5bd546bd9